.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ references:
       at: *working_directory
 
   .unity_linux_image: &unity_linux_image
-    - image: unityci/editor:ubuntu-2021.3.20f1-linux-il2cpp-1.0.1
+    - image: unityci/editor:ubuntu-2021.3.22f1-linux-il2cpp-1.0.1
```

Comment:
Shall we extract this as a variable?

Reply:
As far as I understand, you can do that using it as a CircleCI Environment Variable (like GitHub Secrets). I don't recommend this because it is out of this file, and you lose context.

We should check if CircleCI can do it with a normal variable.

Anyway, to avoid blocking this PR, we can merge and check if that is possible.